Tools/Scripts/webkitpy/w3c/test_downloader.py

```diff
@@ -60,6 +60,7 @@ def __init__(self, repository_directory, host, options):
         self._test_suites = []
 
         self.repository_directory = repository_directory
+        self.rev_parse_hash = "Missing"
```

Comment:
```suggestion
        self.working_copy_head = None
```

Let's use a more descriptive variable name, and let's not use magic strings.

Reply:
done.